docs/conf.py

```diff
@@ -1 +1,6 @@
 from crate.theme.rtd.conf.crate_jdbc import *
+
+# crate.theme sets html_favicon to favicon.png which causes a warning because
+# it should be a .ico and in addition there is no favicon.png in this project
+# so it can't find the file
+html_favicon = None
```

Comment:
shouldn't we put this setting into the theme instead of setting it in every project?